qiskit/pulse/schedule.py

```diff
@@ -81,20 +81,23 @@ class Schedule:
 
       .. code-block:: python
 
+          from qiskit.pulse import Schedule, Gaussian, DriveChannel, Play
           sched = Schedule()
           sched += Play(Gaussian(160, 0.1, 40), DriveChannel(0))
 
     - Appending an instruction shifted in time by a given amount
 
       .. code-block:: python
 
+          from qiskit.pulse import Schedule, Gaussian, DriveChannel, Play
           sched = Schedule()
           sched += Play(Gaussian(160, 0.1, 40), DriveChannel(0)) << 30
 
     - Merge two schedules
 
       .. code-block:: python
 
+          from qiskit.pulse import Schedule, Gaussian, DriveChannel, Play
           sched1 = Schedule()
           sched1 += Play(Gaussian(160, 0.1, 40), DriveChannel(0))
```

Comment:
It doesn't need to be right now, but for the future if we're going to expand this, we should consider what the best user experience is - repeating the same `import`s lots of times feels like it's adding noise rather than clarity to me.

Reply:
Totally agree, I'll remove the second two imports.